portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -24,13 +24,15 @@
   private final String email;
```

Comment:
Does `userId` make `email` redundant? If so, can we get rid of it?

Reply:
The only thing I still need the email for is for generating a link to the email of the user who posted a comment in the comments history, but perhaps for privacy reasons it might not be the greatest idea to include this now that they're not a choice of whether or not to include their email... I'll change this!